imgui/src/widget/color_editors.rs

```diff
@@ -192,6 +192,7 @@ pub struct ColorEdit<'a> {
 
 impl<'a> ColorEdit<'a> {
     /// Constructs a new color editor builder.
+    #[doc(alias = "ColorEdit3", alias = "ColorEdit4")]
```

Comment:
Hahaha, this really shows how much we need to update this. We now have a significantly improved color struct which we'll probably use instead of this in the near future; how odd these aliases look reflects how odd the binding is.